scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -0,0 +1,410 @@
+  input {
+    Array[File] input_vcfs
+    String output_basename
+
+    String gatk_docker = "us.gcr.io/broad-gatk/gatk:4.2.6.1"
+    Int cpu = 1
+    Int memory_mb = 7500
+    Int disk_size_gb = ceil(3*size(input_vcfs, "GiB"))
+  }
+  Int command_mem = memory_mb - 1000
+  Int max_heap = memory_mb - 500
+
+  command <<<
+    set -euo pipefail
+
+    CHR_VCFS_ARG=""
+    for file in ~{sep=' ' input_vcfs}
+    do
+      if [ -s $file ]; then
+        CHR_VCFS_ARG+=" --INPUT $file "
+      fi
+    done
+    echo $CHR_VCFS_ARG
+
+    # --REORDER_INPUT_BY_FIRST_VARIANT means that the vcfs supplied here need not be ordered by location.
+    gatk --java-options "-Xms~{command_mem}m -Xmx~{max_heap}m" \
+      GatherVcfs \
+        --REORDER_INPUT_BY_FIRST_VARIANT \
+        $CHR_VCFS_ARG \
+        --OUTPUT ~{output_basename}.vcf.gz
+
+    tabix ~{output_basename}.vcf.gz
+  >>>
+
+  runtime {
+    docker: gatk_docker
+    cpu: cpu
+    memory: "${memory_mb} MiB"
+    disks: "local-disk ${disk_size_gb} HDD"
+    bootDiskSizeGb: 15
+    preemptible: 1
+  }
+
+  output {
+    File output_vcf = "~{output_basename}.vcf.gz"
+    File output_vcf_index = "~{output_basename}.vcf.gz.tbi"
+  }
+}
+
+task SelectVariants {
+  input {
+    File input_vcf
+    File input_vcf_index
+    String? contig
+    String sample_name
+
+    String output_basename
+
+    String gatk_docker = "us.gcr.io/broad-gatk/gatk:4.2.6.1"
+    Int cpu = 1
+    Int memory_mb = 7500
+    Int disk_size_gb = ceil(2*size(input_vcf, "GiB")) + 50
+  }
+  Int command_mem = memory_mb - 1000
+  Int max_heap = memory_mb - 500
+
+  command <<<
+    gatk --java-options "-Xms~{command_mem}m -Xmx~{max_heap}m" \
+      SelectVariants \
+        -V ~{input_vcf} \
+        ~{"-L " + contig} \
+        --sample-name ~{sample_name} \
+        --select-type-to-exclude NO_VARIATION \
+        -O ~{output_basename}.vcf.gz
+  >>>
+
+  runtime {
+    docker: gatk_docker
+    cpu: cpu
+    memory: "${memory_mb} MiB"
+    disks: "local-disk ${disk_size_gb} HDD"
+    bootDiskSizeGb: 15
+    preemptible: 1
+  }
+
+  output {
+    File output_vcf = "~{output_basename}.vcf.gz"
+    File output_vcf_index = "~{output_basename}.vcf.gz.tbi"
+  }
+}
+
+task Add_AS_MAX_VQSLOD_ToVcf {
+  input {
+    File input_vcf
+    String output_basename
+
+    String docker = "us.gcr.io/broad-dsde-methods/variantstore:ah_var_store_20220425"
+    Int cpu = 1
+    Int memory_mb = 3500
+    Int disk_size_gb = ceil(2*size(input_vcf, "GiB")) + 50
+  }
+
+  command <<<
+    set -e
+
+    python3 /app/add_max_as_vqslod.py ~{input_vcf} > ~{output_basename}.vcf
+  >>>
+  runtime {
+    docker: docker
+    cpu: cpu
+    memory: "${memory_mb} MiB"
+    disks: "local-disk ${disk_size_gb} HDD"
+  }
+
+  output {
+    File output_vcf = "~{output_basename}.vcf"
+  }
+}
+
+task BgzipAndTabix {
+  input {
+    File input_vcf
+    String output_basename
+
+    String docker = "us.gcr.io/broad-gotc-prod/imputation-bcf-vcf:1.0.5-1.10.2-0.1.16-1649948623"
```

Comment:
this container may be better? but fyi bcftools is in the ah_var_store container (and we should probably add tabix if we haven't already)

Reply:
I didn't realize ah_var_store had bgzip (and tabix?), so I went with this one. It's a lot smaller (394 MB vs 4.74 GB for ah_var_store), so the task will run faster. For simple tasks like this it probably makes some sense to use the smaller, but on the other hand standardizing on one docker might make sense too.